ksp_plugin_adapter/differential_slider.cs

```diff
@@ -72,15 +149,22 @@ internal class DifferentialSlider : ScalingRenderer {
 
         if (UnityEngine.GUILayout.Button("0", GUILayoutWidth(1))) {
           value_changed = true;
-          value = 0;
+          // Force a change of value so that any input is discarded.
+          value = zero_value_ + 1;
+          value = zero_value_;
         }
         if (slider_position_ != 0.0) {
           value_changed = true;
+          // Moving the slider doesn't cause a loss of focus so we terminate
+          // input if necessary.
```

Comment:
If I recall correctly, the issue is not that it doesn't cause a loss of focus (it does terminate input if it is another slider), rather that we start changing the value before committing the text input (perhaps because of event order?).

Reply:
I *think* that clicking on the slider/button of the same differential slider doesn't cause a loss of focus at all, and clicking elsewhere does.  I don't feel like experimenting to nail down the exact semantics of Unity, though.